classes/evasys_synchronizer.php

```diff
@@ -397,4 +410,30 @@ public function set_evaluation_period(array $dates) : bool {
 
         return $changed;
     }
+
+    public static function getstandardtimemode($category) {
```

Comment:
Function name should be written with underscores in moodle coding style.
Also phpdoc is missing